metricbeat/module/mysql/query/_meta/fields.yml

```diff
@@ -0,0 +1,9 @@
+- name: query
+  type: group
+  description: >
+    query
+  fields:
+    - name: example
+      type: keyword
```

Comment:
yes, the PR is not complete yet and also not fully functional. Probably best would be to not ship a template in the first step at least.